block-and-parry-demo/Game.gd

```diff
@@ -0,0 +1,55 @@
+extends Node2D
+
+var _freezing := false
+
+onready var _player := $Player
+onready var _enemy := $Enemy
+onready var _camera := $Camera2D
+onready var _tween := $Tween
+
+
+func _ready() -> void:
+	_player.connect("blocked", self, "_on_Player_blocked")
+	_player.connect("hit", self, "_on_Player_hit")
+	_enemy.connect("stunned", self, "_on_Enemy_stunned")
+
+
+func _shake_screen(strength: float, frequency: int, rate: float) -> void:
```

Comment:
The parameters don't do what their name says. Frequency is a number of times per second, but the parameter represents the number of times the camera moves. Rate means the same as frequency, but it's the duration of the individual animations here.